cpp/src/strings/convert/convert_datetime.cu

```diff
@@ -123,7 +123,7 @@ struct format_compiler {
     : format(fmt), d_items(0, stream)
   {
     specifiers.insert(extra_specifiers.begin(), extra_specifiers.end());
-    std::vector<format_item> items;
+    auto items  = cudf::detail::make_empty_host_vector<format_item>(format.length(), stream);
```

Comment:
estimate of the eventual vector size; no need to be exact

Reply:
cool